opentelemetry-sdk/src/metrics/test.md

```diff
@@ -0,0 +1,24 @@
+# Test Coverage for Metric Instruments
+
+Tests are located in [mod.rs](mod.rs)
+
+// TODO: Fill this correctly.
+
+## Sync Instruments
+
+| Test Type                  | Counter (Delta) | Counter (Cumulative) | UpDownCounter (Delta) | UpDownCounter (Cumulative) | Gauge (Delta) | Gauge (Cumulative) | Histogram (Delta) | Histogram (Cumulative) |
+|----------------------------|-----------------|----------------------|----------------------|----------------------------|---------------|--------------------|-------------------|------------------------|
+| Regular aggregation test   | [Yes]         | [Yes]              | [No]              | [No]                    | [No]      | [No]            | [No]           | [No]                |
+| No-attribute test          | [Yes]         | [Yes]              | [No]              | [No]                    | [No]      | [No]            | [No]           | [No]                |
```

Comment:
For implementation of these tests, I think we could combine regular aggregation and no-attribute into a single test to keep the code size for tests less. Check this [test](https://github.com/open-telemetry/opentelemetry-rust/blob/b6ab7287ff787eab93b9150ba1fd5d8a1334f15d/opentelemetry-sdk/src/metrics/mod.rs#L1807-L1809) for example which conditionally emits a zero-attribute measurement along with the regular measurements.